nss-day-cohort-60/national-parks-django-api
# Updated database to correct longitude and latitude values in Park table

# Description

When working on adding a map of park locations (reference: national-parks-client-v2 ticket #6), I realized some of the park values for longitude and latitude were incorrect.  I updated the **national_park_add_data.sql** file to have the correct lat/long values for each park. 

# Steps to Test 

- Run the below commands in terminal

```
git fetch origin vs-parks-map
git checkout vs-parks-map
```

- Delete your database and migration files 
- Run the below commands in terminal 

```
python3 manage.py makemigrations
python3 manage.py migrate
python3 manage.py loaddata users
python3 manage.py loaddata tokens
```

- Then re-run the **national_park_add_data.sql** queries
  - To run, highlight all text in the file and use **cmd + R + R**
- Check the database and ensure the values have been updated
